test/csuite/CMakeLists.txt

```diff
@@ -54,36 +56,40 @@ define_c_test(
         random_directio/main.c
         random_directio/util.c
     DIR_NAME random_directio
-    SMOKE
+    EXEC_SCRIPT ${CMAKE_CURRENT_SOURCE_DIR}/random_directio/smoke.sh
+    ARGUMENTS $<TARGET_FILE:test_random_directio>
     DEPENDS "WT_POSIX"
 )
 
 define_c_test(
     TARGET test_rwlock
     SOURCES rwlock/main.c
     DIR_NAME rwlock
+    ARGUMENTS -h $<SHELL_PATH:$<TARGET_FILE_DIR:test_rwlock>/WT_HOME>
     DEPENDS "WT_POSIX"
 )
 
 define_c_test(
     TARGET test_schema_abort
     SOURCES schema_abort/main.c
     DIR_NAME schema_abort
-    SMOKE
+    EXEC_SCRIPT ${CMAKE_CURRENT_SOURCE_DIR}/schema_abort/smoke.sh
+    ARGUMENTS $<TARGET_FILE:test_schema_abort>
     DEPENDS "WT_POSIX"
 )
 
 define_c_test(
     TARGET test_scope
     SOURCES scope/main.c
     DIR_NAME scope
+    ARGUMENTS -h $<SHELL_PATH:$<TARGET_FILE_DIR:test_scope>/WT_HOME>
```

Comment:
The `EXEC_SCRIPT` might be missing here too

Reply:
It is missing at different places (wt1965_col_efficiency, wt2403_lsm_workload, ...), maybe I misunderstood the way it should be used. Can you help me understand?

Reply:
Same as the earlier comment, `test_scope` isn't executed with a smoke.sh script i.e. the test doesn't have a smoke.sh script (both in CMake and autconf). Rather the test binary is just executed directly.